src/content/docs/new-relic-one/use-new-relic-one/ui-data/maps-in-context.mdx

```diff
@@ -1,19 +1,21 @@
 ---
-title: Maps in context help you understand entity relationships
-metaDescription: Maps in context are available throughout the New Relic One UI to show how your entities relate to one another.
+title: Maps in context display entity relationships
+metaDescription: Maps in context are available in several New Relic UI locations to show how entities relate to one another.
 ---
 
-Maps show how your software entities are connected and deployed, using New Relic's auto-discovered and user-defined relationships for topology.
+Some New Relic features, like distributed tracing and Applied Intelligence, display topological maps that show relationships between apps, hosts, services, and other [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic) to aid in diagnosing and correcting issues. 
 
-Various New Relic features, such as AIOps Issues and Distributed Tracing, include maps in context to help you quickly assess, diagnose, and correct issues by illustrating how your complex software systems are composed. The map's content and presentation vary depending on your use case, while some features are always available.
+## Map display [#map-display]
 
-## Common map view
+The exact display of these maps depends on the New Relic feature in which they're located, but here we'll describe some common visual elements. 
 
-The typical map view shows you entities, represented as hexagons, and their relationships, represented as lines with arrows indicating call direction.
+The typical map view uses hexagon icons to represent [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic). The relationships between entities are represented as lines, with arrows indicating call direction. 
```

Comment:
The comma isn't necessary but I'll let you decide since it doesn't add or subtract from what you're trying to say. :)

Reply:
I'm confused about github formatting; which comma is unnecessary?